Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/sheets/person/ContactDiaryPersonBottomSheetDialogViewModel.kt

```diff
@@ -21,22 +21,33 @@ class ContactDiaryPersonBottomSheetDialogViewModel @AssistedInject constructor(
     @Assisted private val addedAt: String?,
     private val contactDiaryRepository: ContactDiaryRepository
 ) : CWAViewModel(dispatcherProvider = dispatcherProvider) {
+
     private val text = MutableStateFlow("")
 
     val isValid = text.map {
-        it.isNotEmpty() && it.length <= MAX_PERSON_NAME_LENGTH
+        it.isNotEmpty()
     }.asLiveData()
 
     val shouldClose = SingleLiveEvent<Unit>()
 
+    private val formattedName: String
```

Comment:
Duplicate code - could be an extension function instead.

Reply:
Extension of what?

Reply:
of String ... e.g. `fun String.trimNoneWitespaceWithLimit(limit: Integer)`

Reply:
done